README.md

```diff
@@ -546,6 +546,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [cry](https://github.com/elorest/cry) - Ability to execute crystal code in a fashion similar to Ruby's pry edit
  * [Crystal [ANN]](https://crystal-ann.com) - Announce new project, blog post, version update or any other Crystal work
  * [crystalshards.herokuapp.com](https://crystalshards.herokuapp.com/), [crystalshards.xyz](http://crystalshards.xyz/) - Web services that list all available Crystal shards
+ * [Crystular](http://www.crystular.org) - regular expression tester ([GitHub repo](https://github.com/Selleo/crystular))
```

Comment:
Lets start the description with an uppercase letter and remove the Github link (you have it on the site already)

Reply:
updated